stac_fastapi/types/stac_fastapi/types/stac.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List, Optional, TypedDict, Union
+
+NumType = Union[float, int]
+
+
+class LandingPage(TypedDict):
+    stac_version: str
+    id: str
+    title: str
+    description: str
+    conformsTo: List[str]
+    links: List[Dict[str, Any]]
+
+
+class Conformance(TypedDict):
+    conformsTo: List[str]
+
+
+class Catalog(TypedDict):
+    type: str
+    stac_version: str
+    stac_extensions: Optional[List[str]]
+    id: str
+    title: Optional[str]
+    description: str
+    links: List[Dict[str, Any]]
+
+
+class Collection(Catalog):
```

Comment:
shouldn't use inheritance here

Reply:
Does `TypedDict` act funny down the inheritance chain?

Reply:
No but it doesn't support `isinstance` or `issubclass` checks so I think the only point is DRY which seems unnecessary for only a few fields.